### DISCUSSION TITLE
How can i use tailwind css in codeigniter 3

### DISCUSSION THREAD

**Discussion by jaipalsin** [tailwindlabs/tailwindcss#12330] [Help] [answered]
I'm designing an entire website in CodeIgniter 3 and I want to use Tailwind CSS for it. but Tailwind CSS has not provided any guidance for the installation & configuration of Codeigniter but in tailwind CSS Here is a CDN that is not recommended by Tailwind CSS.I want to use tailwind CSS in Codeigniter in genuine ways.

**Comment by wongjn** [comment, marked as answer]
You could consider using the [Tailwind CLI](https://tailwindcss.com/docs/installation) if you do not have any other Node.js-based build process. Ensure the [`content`](https://tailwindcss.com/docs/content-configuration) file globs cover your template files and then have the output CSS of the `--output` flag be located where you can load it from the CodeIgniter framework.

**Reply by jaipalsin** [reply]
Thanks for this help

**Reply by codedrian** [reply]
Hiya. What's up? Did you used tailwind cli? I'm about to use tailwind with codeigniter right now.
